codewof/programming/content/en/space-cargo/question.md

```diff
@@ -0,0 +1,10 @@
+# Space Cargo
+
+Make a function `eject_cargo(cargo, hull_capacity)` that takes two arguments:
+
+- `cargo`, a Python list of strings representing items stored in the hull of your spaceship. Each item weighs 1 unit.
+- `hull_capacity`, an integer representing how much space you have on board.
+
+It should pop items from the end of `cargo` and then **print** `'Ejected {item}'`. 
+It should do this until the spaceship is no longer too full.
```

Comment:
```suggestion
If the spaceship has more cargo than hull capacity, it should pop items from the end of `cargo` and **print** `'Ejected {item}'`, until the spaceship can carry the remaining items in it's hull.
```

Reordering this make it a bit easier to read, starting with the condition, and rewording the okay condition.